cpu-miner.c

```diff
@@ -741,10 +744,10 @@ static void *miner_thread(void *userdata)
 		gettimeofday(&tv_start, NULL);
 
 		max64 = HASH_SCANTIME * thr_hashrates[thr_id];
-		if (max64 == 0) max64 = 0xffffff;
+		if (max64 == 0) max64 = 0xfffff;
```

Comment:
For me this was also supposed to be `0xffffff` and not `0xfffff`.

What do you think?

Reply:
`cpuminer` estimates the amount of nonce values it should try based on its hashrate.
On the first iteration it will have no hashrate estimate, then `max64 = HASH_SCANTIME * thr_hashrates[thr_id];` will be `0`. 

In this case we have to set a first value for it. Scan `0xfffff == 2 ** 20 - 1 == 1048575` hashes seems fine.

Reply:
Ok, I agree, it's fine